app/Helpers/PolygonGeometryHelper.php

```diff
@@ -58,4 +58,13 @@ public static function updateProjectCentroidFromPolygon($polygonGeometry)
             Log::error('Error updating project centroid: ' . $e->getMessage());
         }
     }
+
+    public static function getPolygonsWithNames(array $uuids)
```

Comment:
I don't think this method is going to be reused anytime soon the way it is, my suggestion would be to call this as `getPolygonsProjection` and pass parameters `$fields` that will be the current array of fields that you want to project your site polygon

Reply:
Also I noted that your method does not take the `is_active` scope or filter into account. Please confirm that this is the intended behavior

Reply:
Changed to be more usefull

Reply:
also added is_active